charts/consul/templates/cni-clusterrole.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ template "consul.fullname" . }}-cni
-  namespace: {{ .Release.Namespace }}
```

Comment:
I'm not 100% sure that it makes sense to put the `ClusterRole` or `ClusterRoleBinding` in the new `connectInject.cni.namespace` namespace, I dont believe we expose that in other clusterole/bindings, however I *think* it looks nice to have everything related to CNI in the same namespace.

Reply:
I agree. It should probably be in the same namespace.